### DISCUSSION TITLE
full width datepicker

### DISCUSSION THREAD

**Discussion by AndrewDeminWeb** [henninghall/react-native-date-picker#834] [General]
Hey, everybody.
Saw a pull request that fixed the inability to change the width (make it 100%), now I want to do it myself, but the width does not change.
![react native date picker](https://github.com/henninghall/react-native-date-picker/assets/104236078/5ce13c90-1ae6-4ba6-b8b9-458d90138706)

**Comment by Ellenyyyy** [comment]
style={{ width: null }}

**Reply by AndrewDeminWeb** [reply]
He's disappeared altogether

**Comment by vorn-tovtrip** [comment]
it doesnt work i want it to take full width and have space around

**Comment by Alaaa1** [comment]
did anyone figure a work around for this 🤔
